frontend/tests/sprintboard/boardSpec.js

```diff
@@ -36,7 +36,6 @@ test('Sprint board renders the correct number of columns', (assert) => {
   // this is a pretty stupid test, because I can not assign
   // an arbtrary number of columns during the test; it currently checks
   // that the number of columns is the same as in src/mock-data/sprint-columns.js file
-  assert.equal(sprintBoard.props.children.length, 4,
-    'Sprint board should render four sprint columns');
+  assert.equal(sprintBoard.props.children[1].length, 4, 'Sprint board should render four sprint columns');
```

Comment:
Adding another component to the sprint board changed `this.props.children` from referencing the `sprintColumns` component directly to an array that includes `CreateTask` and `sprintColumns`.